### ISSUE TITLE
geting classification model train() function output without batch size

### ISSUE THREAD

**Issue by azizcu** [ThilinaRajapakse/simpletransformers#1076]
I want to use classification model train() function output for the last hidden layer but I can not convert it like the predict() function all_layer_hidden_states[11](set config, output_hidden_states=True) . For predict function output shape is ( 1517, 128, 768 ) #number of input test sentence =1517

outputs = model(**inputs) # classification model train() output
last_layer = outputs[2][11]
shape = 8, 128, 768 #bert-base-uncased model
here 8 = batch size but I want it without batch size like shape = 1517, 128, 768 #number of input train sentence =1517

layer_hidden_states = outputs[2][1:]
all_layer_hidden_states = np.array([state.detach().cpu().numpy() for state in layer_hidden_states])
I try to convert it like that way but not worked.                 

please help. Thanks for your continuous co-operation

**Comment by ThilinaRajapakse** [comment]
You should be able to just reshape the tensors with `reshape`. (See torch docs)
